libs/libarchfpga/src/echo_arch.cpp

```diff
@@ -232,9 +232,13 @@ void PrintArchInfo(FILE* Echo, const t_arch* arch) {
     }
 
     fprintf(Echo, "\tInput Connect Block Switch Name Within a Same Die: %s\n", arch->ipin_cblock_switch_name[0].c_str());
```

Comment:
I think it's worth commenting the [0] and [1] indices: [0] is the ipin_cblock_switch_name for within die and [1] is between dice?  This might be more clear with constexpr that gives names to 0 and 1 for the lookups.

Reply:
@vaughnbetz fixed it